src/desktop/src/ui/views/onboarding/Welcome.js

```diff
@@ -49,6 +49,10 @@ class Welcome extends React.PureComponent {
         scrollEnd: false,
     };
 
+    componentDidMount() {
+        this.changeLanguage(detectLocale(navigator.language));
```

Comment:
`detectLocale` is missing a fallback to an existing locale if `navigator.language` contains a locale that is not supported by Trinity.